src/main/php/org/bovigo/vfs/vfsStreamFile.php

```diff
@@ -393,4 +397,19 @@ public function hasExclusiveLock($resource = null): bool
 
         return null !== $this->exclusiveLock;
     }
+
+    /**
+     * renames the content
+     *
+     * @param   string  $newName
+     * @throws  vfsStreamException
+     */
+    public function rename(string $newName)
+    {
+        if (strstr($newName, '/') !== false) {
```

Comment:
Given that this makes `vfsStreamFile::rename()` and `vfsStreamDirectory::rename()` virtually identitical, could this check be moved to `vfsStreamAbstractContent::rename()` instead?

Reply:
It could. I originally didn't because I wasn't trying to change the behavior of `vfsStreamBlock`, but I just noticed it extends `vfsStreamFile` and not the abstract so now all three share the same behavior. Definitely makes more sense moving it to the abstract.